src/hooks/useWalletConnection.ts

```diff
@@ -1,12 +1,13 @@
 import { walletApi } from 'api'
-import { useState, useEffect } from 'react'
+import { useEffect } from 'react'
 import { WalletInfo, Command } from 'types'
+import useSafeState from './useSafeState'
 
 export const useWalletConnection = (): WalletInfo => {
-  const [walletInfo, setWalletInfo] = useState<WalletInfo>({ isConnected: false })
+  const [walletInfo, setWalletInfo] = useSafeState<WalletInfo>({ isConnected: false })
 
   useEffect((): Command => {
     return walletApi.addOnChangeWalletInfo(setWalletInfo, true)
```

Comment:
when does `setWalletInfo` change? I mean, if that ever happens, shouldn't we also desubscribe?

Reply:
Doesn't change, linter wants it there. Either add or disable the rule.